pkg/settings/setting_test.go

```diff
@@ -0,0 +1,35 @@
+package settings
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsRelease(t *testing.T) {
+	inputs := map[string]bool{
+		"dev":         false,
+		"master-head": false,
+		"master":      false,
+		"v2.5.2":      true,
+		"v2":          true,
+		"v2.0":        true,
+		"v2.x":        true,
+		"v2.5-head":   false,
+		"2.5":         false,
+		"2.5-head":    false,
+	}
+	for key, value := range inputs {
+		err := ServerVersion.Set(key)
+		if err != nil {
+			t.Errorf("Encountered error while setting temp version: %v\n", err)
+		}
+
+		// We use "!assert.Equal" here instead of "assert.NotEqual" because we want the testing harness to know that
+		// the desired outcome is for both objects to be equal.
+		result := IsRelease()
+		if !assert.Equal(t, value, result) {
+			t.Errorf("Expected value [%t] for key [%s]. Got value [%t]", value, key, result)
+		}
```

Comment:
Using both `assert` and `t` is redundant
```suggestion
		assert.Equal(t, value, result, fmt.Sprintf("Expected value [%t] for key [%s]. Got value [%t]", value, key, result))
```